app/src/main/java/com/edulectronics/tinycircuit/Models/Scenarios/ImplementedScenarios/NonExistingScenario.java

```diff
@@ -0,0 +1,15 @@
+package com.edulectronics.tinycircuit.Models.Scenarios.ImplementedScenarios;
+
+import com.edulectronics.tinycircuit.Models.Scenarios.DesignScenario;
+import com.edulectronics.tinycircuit.R;
+
+/**
+ * Created by Jesper on 1/12/2017.
+ */
+
+public class NonExistingScenario extends DesignScenario{
```

Comment:
Wat is een NonExtistingScenario? Waar gebruik je dit voor? Graag commentaar boven de klasse zetten